### ISSUE TITLE
macOS 10.15 Catalina support

### ISSUE THREAD

**Issue by MattIPv4** [electron/electron#18660]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 5.0.2
* **Operating System:**
  * macOS 10.15 Catalina

Hi all, sorry for not following a template format fully - I couldn’t decide what would be most appropriate for this.

Is there a roadmap or similar available for official support of macOS 10.15 Catalina?

**Comment by MarshallOfSound** [comment]
@MattIPv4 The Catalina beta was released 2 days ago 😄 The general policy will be "let's find out what (if anything) is broken".  As developers update their machines to the developer preview we'll get bug reports for things that are broken, Chromium will also be receiving reports in parallel if things are broken.  Personally I don't have a machine just lying around that I can risk the beta on but as it stabilizes we'll get reports and be able to more testing.

TLDR: No roadmap, as things break we'll obviously try to fix them 😄
